website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
+You need to have a running Kubernetes cluster to set up PostgreSQL with DocumentDB extension on Kubernetes.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.102.0-ferretdb-2.1.0`)
+to ensure consistency across deployments.
+For more information on the best FerretDB image to use, see the [DocumentDB release notes](https://github.com/FerretDB/documentdb/releases/).
+:::
+
+The following `postgres.yaml` file creates a PostgreSQL instance with the DocumentDB extension:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: postgres
+  labels:
+    app: postgres
+spec:
+  containers:
+    - name: postgres
+      image: ghcr.io/ferretdb/postgres-documentdb:17-0.102.0-ferretdb-2.1.0
```

Comment:
This is version does not exit. Could you please make sure to use the working version please and also test the change if config is modified please. I believe `17-0.102.0-ferretdb-2.0.0`

Reply:
FYI think next version will be `17-0.103.0-ferretdb-2.1.0` and not `17-0.102.0-ferretdb-2.1.0`, so I think it's important to write a documentation against what current version :hugs:

Reply:
Updated to the working version now. Further changes will be made later for current/next docs version.

Reply:
Please see https://github.com/FerretDB/FerretDB/blob/main/.github/RELEASE_CHECKLIST.md. We should specify the anticipated version in the current documentation.